ceph_iscsi_config/gateway_setting.py

```diff
@@ -171,7 +171,9 @@ def normalize(self, raw_val):
     "pool": StrSetting("pool", "rbd"),
     "cluster_client_name": StrSetting("cluster_client_name", "client.admin"),
     "time_out": IntSetting("time_out", 1, 600, 30),
-    "api_host": StrSetting("api_host", "::"),
+    # if the api_host has a colon, you must wrap it with square brackets.
+    # i.e:[::]
+    "api_host": StrSetting("api_host", "localhost"),
     "api_port": IntSetting("api_port", 1, 65535, 5000),
     "api_secure": BoolSetting("api_secure", True),
     "api_ssl_verify": BoolSetting("api_ssl_verify", False),
```

Comment:
You will need to add api_host to exclude_from_hash in ceph_iscsi_config/setting.py, because that value can be different on each node.

Reply:
ok i add it